src/Balloon.scala

```diff
@@ -0,0 +1,13 @@
+
+class Balloon{
+//minimal definition of a balloon. Method call sings 99 red balloons by nena
```

Comment:
Okay, but make use of the common Javadoc syntax by putting the comment above the class definition.
And use the /** ..*/ syntax.